app.js

```diff
@@ -23,3 +23,24 @@ app.get('/', (req, res) => {
 });
 
 app.listen(3000, () => console.log('🏃‍ on port 3000'));
+app.get('/beers', (req, res) => {
+  const beers = punkAPI.getBeers()
+
+  beers.then(beersArr => {
+    let newArr = beersArr
```

Comment:
API calls usually are pretty direct in what they do, but if it works for you to store it for better understanding be careful with how you name it; be as specific as you can to have it structured and easy to read.